docs/customizing.md

```diff
@@ -0,0 +1,195 @@
+Customizing Your Pulp Installation
+==================================
+
+Background info on Ansible variables
+------------------------------------
+
+Because the pulp installer is based on Ansible it inherits 2 Ansible designs:
+
+1. Variables can be set in many different ways by the user.
+2. Each role has its own set of variables.
+
+This section (and the docs in general) will not cover every possible way of [setting variables](https://docs.ansible.com/ansible/latest/user_guide/playbooks_variables.html#where-to-set-variables), but instead cover the
+simplest way: In the playbook itself.
+
+This section will not cover every single variable. They are all
```

Comment:
```suggestion
This section intends to demonstrate how to use Ansible variables to configure Pulp rather than listing every variable. Documentation for each Ansible variable can be found in each Ansible role's README file.
```